VaticanBAV/et/BAVet120.xml

```diff
@@ -28,148 +28,247 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-  <persName ref="PRS13009Gezot">ግዛት</persName>፨
-  <persName ref="PRS13005WalattaSeyon"> ወለተ፡ ጽዮን</persName>፨ <persName ref="PRS13010Warqe">ወርቄን፡</persName>
-  ስመ፡ ጥምቀቱ፡ ወልደ፡ ሚካኤል፤ እረፍቱኒ፡
-  እመ፡ |ተ| ፱ለሚያዝያ፡ ወብእሲቱ፡ <persName ref="PRS13011WalattaGabriel">ወለተ፡ ገብርኤል፡</persName>
-  እቴ፡ ናት፡ ዕረፍታ፡ አመ፡ ፳ወ፱ለነሐሴ፨
-  ወመቃብረ፡ ክልዔሆሙ፡ <placeName ref="LOC7343SirabaSeyon">ሲራባ፡ ጽዮን</placeName>፨ ዝኒ፡ ለተዝካረ፡ ስሞሙ፡ ተጽሐፈ፨
-  <date calendar="grace" when="1915" when-custom="1923" evidence="internal">
-    በ፲ወ፱፻፲ወ፭ዓመተ፡ ምሕረት፨</date></q>
-</item>
-
-<item xml:id="a2">
-<locus target="#89"></locus>
-<desc type="Supplication">ኦእግዚአ፡ ዕቀበኒ፡ ወአድኅነኒ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ሊተ፡
-  ለገብርከ፡ ገብረ፡ ሥላሴ፡</desc>
-</item>
-</list>
-<list>
-<item xml:id="e1">
-<locus from="1" to="2v"></locus>
-<desc>Pen trials</desc>
-
-</item>
-</list>
-</additions>
-<bindingDesc>
-<binding contemporary="true" xml:id="binding">
-<decoNote xml:id="b1">Two wooden boards.</decoNote>
-  <decoNote xml:id="b2" type="bindingMaterial"><material key="wood">
-</material>
-  </decoNote>
-</binding>
-</bindingDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#89"/>
+                                    <desc xml:lang="gez" type="Supplication">Supplication mentioning <persName role="donor" ref="PRS12978GabraSelassie">Gabra Śǝllāse</persName></desc>
+                                        <q>
+                                            ኦእግዚአ፡ ዕቀበኒ፡ ወአድኅነኒ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ሊተ፡
+                                            ለገብርከ፡ ገብረ፡ ሥላሴ፡
+                                        </q>
+                                </item>
+                            </list>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus from="1" to="2v"/>
+                                    <desc>Pen trials.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1">Two wooden boards.</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="wood">
+                                </material>
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
                     </physDesc>
+                    
+                    <history>
+                        <origin>
+                            <origDate notBefore="1922" notAfter="1923">
+                         According to <ref target="#colophon1"/>, the manuscript was written in the year of mercy 
+                                <date calendar="ethiopian">1915 EC</date>, i.e. <date>1922/23 CE</date>.
```

Comment:
I think, it is recommendable to mark up dates to make it more filterable, i.e.:
 `<date notBefore="1922" notAfter="1923">1922/1923 CE</date>` or even more concisely:
`<date notBefore="1922" notAfter="1923" calendar="ethiopian" when-custom="1915">1915 EC</date>`